src/EPR.RegulatorService.Frontend.Web/Resources/Views/Shared/Partials/_RegistrationResubmissionDetails.en.resx

```diff
@@ -63,4 +162,7 @@
   <data name="SummaryList.SubTitle.PreviousReasonForRejection" xml:space="preserve">
     <value>Reason for rejection of previous submission</value>
   </data>
-</root>
+  <data name="SummayList.ReSubmissionPeriod" xml:space="preserve">
```

Comment:
this should be SummaryList instead of SummayList

since this text appears at the end of the sentence, and has a lower case letter, I would recommend calling it:

SummaryList.SubmissionPeriodSuffix

Reply:
Thanks Simon, its updated now.